src/widget/whotcuebutton.cpp

```diff
@@ -89,6 +90,10 @@ void WHotcueButton::mousePressEvent(QMouseEvent* e) {
             if (!pHotCue) {
                 return;
             }
+            if (QGuiApplication::keyboardModifiers().testFlag(Qt::ShiftModifier)) {
+                pTrack->removeCue(pHotCue);
```

Comment:
How about implementing this on the scrolling and overview waveforms too?

Reply:
I implemented it in the overview. I implemented it shortly in the scrolling but then came the conclusion that it would make more sense to use this to move the cue point along the axis since this is high resolution and that would be cool if they could be moved.